.ocamlformat

```diff
@@ -0,0 +1,6 @@
+profile=ocamlformat
```

Comment:
Can you tell what is wrong with the conventional profile?

Reply:
Umm, I guess nothing is wrong, after this is a matter of taste; the reason for this choice is that in < 0.12 ocamlformat this was the default so quite a few headers of the codebase for example are close to that style already. I like some choices in that style indeed, and for example the micromega reformatting was using it as it was done in 0.11.

Reply:
That being said if `conventional` + tweaking for example indentation of function signatures is better we can use that.

Reply:
But micromega reformatting did not use the set of options you give here. For instance, `sequence-style` was obviously set to the previous default, and `exp-grouping` to `parens`.

By "wrong" I did not mean "objectively wrong" but what exactly does it change and what you do not like. The issue with using this profile is that it is much less clear what it does compared to just using the default profile and tweaking a few options. It is my impression that this profile was provided mostly to simplify backward-compatibility and should be avoided for projects newly using `ocamlformat`.

Reply:
I'm not sure the `ocamlformat` profile should be avoided, it is a profile like the others, where did you see that?

I think it is better to be concrete with options instead of letting them implicit in the profile, this will make upgrades more robust and helps document the profile a bit.

Reply:
I mean it is not deprecated so for sure we can use it, but they made some efforts to provide the new `conventional` profile that is intended to better match the formatting practice in the OCaml ecosystem. So, if projects like Coq, that didn't use `ocamlformat` before, and are using quite conventional formatting, choose not to use this `conventional` profile, that is kind of big FAIL for them. I am OK to make the profile explicit as this will indeed be more robust.

Reply:
`conventional` is almost "legacy", in the sense that a lot of more modern OCaml code [for example JS packages] use a pretty different style. Coq is currently a mix, as I've tried to move towards newer styles [as I prefer them]

Coq is not going to fit any style as it is very diverse so I'd just pick the style we like.

Reply:
OK

Reply:
Now that's going to be a hard choice, what style do we like ? XD